mycroft/res/text/eu-eu/message_synching.clock.dialog

```diff
@@ -0,0 +1 @@
+< < < SINKRONIZATZEN < < <
```

Comment:
Should this be cleaned to alphas?

Reply:
no, this is what is sent to the mk1 faceplate, the < are part o the scrolling message and added on purpose